### DISCUSSION TITLE
Unable to connect Stripe on my account...

### DISCUSSION THREAD

**Discussion by UZuccaro** [swellstores/community#295] [Help]
Hi. Everytime I try to connect Swell to my account I recieve a console error and the Stripe dialog (that is the one where we can connect Stripe to Swell) never pops out. I'm stuck with this project, because I need to go live as soon as possibile with the store, but without the Stripe connection we cannot get the payments for the orders (it's the only payment gateway that we actually need). I hope you can help me.

Store: https://filagosto.swell.store/

<img width="1264" alt="Screenshot 2023-07-03 alle 09 20 43" src="https://github.com/swellstores/community/assets/5860049/b1a514e7-2a4a-44ca-98ea-fcf95f96fd49">

**Comment by logeshswell** [comment]
@UZuccaro I confirm that this is a bug in our admin dashboard. I will create a bug ticket for faster resolution.
